nftest/syslog.py

```diff
@@ -0,0 +1,96 @@
+"""
+Module for a system to capture Nextflow logs via syslog.
+"""
+import logging
+import re
+import socketserver
+
+
+LEVELS = [
+    logging.CRITICAL,   # 0 = Emergency
+    logging.CRITICAL,   # 1 = Alert
+    logging.CRITICAL,   # 2 = Critical
+    logging.ERROR,      # 3 = Error
+    logging.WARNING,    # 4 = Warning
+    logging.INFO,       # 5 = Notice
+    logging.INFO,       # 6 = Info
+    logging.DEBUG,      # 7 = Debug
+]
```

Comment:
Is this intended to map logging levels to another priority ranking?

Reply:
It's mapping from [syslog severity levels](https://en.wikipedia.org/wiki/Syslog#Severity_level) _to_ logging levels. The syslog severity is embedded in the message, and I use it as the index into this array to get the appropriate python logging level.

Reply:
Here's the relevant code chunk (and some more details about [priority](https://datatracker.ietf.org/doc/html/rfc5424#section-6.2.1)):

https://github.com/uclahs-cds/tool-NFTest/blob/f27ed68ee4263cd823d6da89493e334356c929c0/nftest/syslog.py#L42-L46